mentalab-explore/pop_loadfile.m

```diff
@@ -73,3 +80,19 @@ function checkFolderContents(filename, directory) % Will be CSV file
          error(['---> Selected directory does not contain: ' name '_Marker.csv.'])
     end
 end
+
+
+function channelNameList = requestChannelLabels(EEG)
+    if (EEG.nbchan < 5)
+        prompt = {'Reference:','Channel 1:','Channel 2:','Channel 3:'};
+        definput = {'Ref', 'Ch1', 'Ch2', 'Ch3'};
```

Comment:
There are two issues here:

- The channel names should start with Ch1. All the channels are measured against the REF electrode and there is no separate REF or GND signal. For example, we will have Ch1, Ch2, Ch3 and Ch4 in a 4-channel device.
- If some channels are already disabled, the above `if else` statement won't be correct. We should check the device info packet for the active channels. An example which the current approach fails is: Ch1, Ch3, Ch5 and Ch7 are disabled by the user. The plugin should show the other channels' name in the pop up window but instead it shows ch1 to ch4 now.